test/charlatan_http_client_adapter_test.dart

```diff
@@ -116,18 +116,6 @@ DELETE /users
       expect(result.data, Uint8List(1));
     });
 
-    test('it provides the path parameters to the response builder', () async {
-      charlatan.whenGet(
-        '/users/{id}/{other}',
-        (request) => {'pathParameters': request.pathParameters},
```

Comment:
If we really want to keep this functionality, I think it's possible. It's just more effort than I wanted to invest here. I _think_ the approach I'd take is to create a subclass of the `CharlatanHttpRequest` type that can be enhanced with the matched `CharlatanResponseDefinition` and it can use the underlying dio request and the definition to extract the path parameters. I started to implement that and got hung up on... something... that I can't remember. 

I thought this was a cute feature when I added it originally, and I still really like it, but I didn't want to work out the kinks of making it continue to work. Open to someone tackling it though if we can find a clean solution. I just don't want to carry around a gnarly implementation for a feature that I'm not sure folks are clamoring for.

Reply:
Since we likely are one of the heaviest users of Charlatan and have zero usages of that API (that I can find at least), I'd agree let's not force it now. If it ends up being missed, we can investigate supporting it again in a future version and folks that really need it can wait for that version.